keps/sig-instrumentation/3077-contextual-logging/README.md

```diff
@@ -507,6 +509,37 @@ The logcheck static code analysis tool will warn about code in Kubernetes which
 calls the underlying functions directly. Once the feature gate is no longer needed,
 a global search/replace can remove the usage of these wrapper functions again.
 
+Because the feature gate is off during alpha, log calls have to repeat
+important key/value pairs even if those also got passed to `WithValues`:
+
+```
+logger := logger.WithValues("pod", klog.KObj(pod))
+...
+logger.Info("Processing", "pod", klog.KObj(pod))
+...
+logger.Info("Done", "pod", klog.KObj(pod))
+```
+
+Starting with beta, the feature gate will be enabled and code can be written
```

Comment:
Does this require another re-write during beta, or at this point in time we'll let it be, just in case someone decides to turn the feature off, thus falling back to previous mechanism, which as described above requires that duplication?

Reply:
I need to update this section. We decided to keep the duplication as long as the feature still can be turned off, which is the opposite of what I was proposing here.